File: api/src/AutoriaClone/AutoriaClone.Api/Application/Responses/User/UserResponseDto.cs
using AutoriaClone.Domain.Aggregates.ValueObjects.Address;

namespace AutoriaClone.Api.Application.Responses.User;

public record UserResponseDto(
    string Email,
    string? FirstName,
    string? LastName,
    string? PhoneNumber,
    string? TelegramUserName,
    string? WebSiteUrl,
    AddressValueObject? Address);

File: mobile/composeApp/src/commonMain/kotlin/com/vpch/autoriamobile/features/data/user/dto/UserResponseDto.kt
package com.vpch.autoriamobile.features.data.user.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class UserResponseDto(
    @SerialName("email") val email: String,
    @SerialName("firstName") val firstName: String?,
    @SerialName("lastName") val lastName: String?,
    @SerialName("phoneNumber") val phoneNumber: String?,
    @SerialName("telegramUserName") val telegramUserName: String?
)
